pkgs/build-support/rust/test/import-cargo-lock/git-dependency-rev-non-workspace-nested-crate/Cargo.lock

```diff
@@ -0,0 +1,638 @@
+# This file is automatically @generated by Cargo.
```

Comment:
This seems like a rather big example to test this case. Can't we build something more simple?

Reply:
i didn't build the example, it's the real-world use-case that surfaced the bug for me. can you explain your concern further? i can't think of any practical downside.

Reply:
My only concern is complexity induced by the test case and runtime on hydra but those aren't really blockers.